src/support/components/ConfirmationOverlay.tsx

```diff
@@ -80,12 +80,14 @@ const ConfirmationOverlay: FC<OwnProps> = () => {
       <Overlay.Header
         testID="confirmation-overlay-header"
         title={
-          params.type === 'PAYG' ? 'Contact Support' : 'Feedback & Questions'
+          params.type === 'contract'
+            ? 'Contact Support'
+            : 'Feedback & Questions'
         }
```

Comment:
I see that you can still end up in this overlay in other account types from FeedbackQuestionsOverlay. What path does the user need to follow to end up in FeedbackQuestionsOverlay? I don't see it mentioned as a reference anywhere other than the overlay controller.

Reply:
Hm, good point, doesn't look like there's another way to get to this. I think it may be a holdover from when the form had the option to submit feedback without having a severity level assigned. I'll remove it.

Reply:
updated

Reply:
Taking a look in a moment